README.md

```diff
@@ -138,7 +138,7 @@ Pass `-d` or `--debug` to `invoke less` to create a CSS sourcemap, useful for de
 
 ```shell
 $ cd <path to repo>
-$ python -m nbviewer --debug --no-cache
+$ python -m nbviewer --debug --no-cache --host=127.0.0.1
```

Comment:
I think listening on 0.0.0.0 by default is a security problem. Well, here on macos, it present me from doing so without root access so It's ok-ish, but i'd like to fix it.